pymc/tests/test_distributions.py

```diff
@@ -2112,6 +2113,32 @@ def test_dirichlet(self, n):
             dirichlet_logpdf,
         )
 
+    @pytest.mark.parametrize(
+        "value,alpha,K,logp",
+        [
+            (np.array([5, 4, 3, 2, 1]) / 15, 0.5, 4, 2.2057773112876893),
+            (np.tile(1, 13) / 13, 2, 12, 13.286898065112881),
+            (np.array([0.001] * 10 + [0.99]), 0.1, 10, -20.66907735582068),
+            (np.append(0.5 ** np.arange(1, 20), 0.5 ** 20), 5, 19, 92.59518981534681),
+        ],
+    )
+    def test_stickbreakingweights(self, value, alpha, K, logp):
+        with Model() as model:
+            StickBreakingWeights("sbw", alpha=alpha, K=K)
+        pt = {"sbw": value}
+        assert_almost_equal(
```

Comment:
I am getting a `TypeError: Unknown input or state: sbw.` error here... Am I supposed to name things different or replace `pt` by `pt = {"sbw_simplex__": value}`? If I do that, then I obtained `AssertionError` but I'm not sure if that's the correct approach...

Reply:
You have to disable transform, with `transform=None`, or else your reference logp has to account for the Jacobian term as well.

Reply:
This works well, thanks!